blog/2015-03-31-Eclipse_Docker_Tooling.adoc

```diff
@@ -0,0 +1,90 @@
+= Upcoming Docker Tooling for Eclipse
+:page-layout: blog
+:page-author: xcoulon
+:page-tags: [jbosstools, docker, eclipse]
+
+We are currently busy working on Docker tooling for Eclipse and even though the
+code is not public yet (more details on that at the end of this blog entry), we thought it
+would be interesting to share our progress on the project.
+
+The Docker tooling is aimed at providing in Eclipse the same level features as
+users have with the command-line interface.
+We rely on the https://github.com/spotify/docker-client[open source Docker client
+developped by Spotify] to handle the low-level communication with the Docker daemons
+and we provide views and wizards to setup connections and manage images
+and containers.
+
+== Docker Explorer
+
+The *Docker Explorer* is a tree view that handles multiple connections and provides users
+with quick overview of the images and containers.
+
+image::images/docker_tooling_explorer_view.png[Docker Explorer view]
+
+Built-in filters can show/hide intermediate and 'dangling' images as well as
+stopped containers.
+
+A wizard lets the user establish a new connection. This wizard can detect default
+settings if the user's machine runs Docker natively or in a VM using Boot2Docker.
+Both Unix sockets on Linux machines and the REST API on other OSes are detected and supported.
+
+image::images/docker_tooling_connection_wizard.png[Connection wizard]
+
+
+== Managing Docker Images
+
+The *Docker Images view* lists all images in the Docker host selected in the
+Docker Explorer view. This view allows user to manage images, including:
+
+- Pulling images from the https://registry.hub.docker.com/[Docker Hub Registry] (other registries will be supported as well)
+- Uploading images to the https://registry.hub.docker.com/[Docker Hub Registry]
+- Building images from a Docker files
+- Running an image
+
+image::images/docker_tooling_images_view.png[Docker Images view]
+
+== Managing Docker Containers
+
+The *Docker Containers* view lets the user manage her containers. The view toolbar
+provides commands to start, stop, pause, unpause, attach and kill containers.
+
+image::images/docker_tooling_containers_view.png[Docker Containers view]
+
+This view also provides a filter to show/hide stopped containers.
+
+== Info and Inspect on Images and Containers
+
+We also integrate with the Eclipse Properties view to provide users with info and 'inspect'
+data about a selected container or image.
+
+image::images/docker_tooling_properties_view.png[Properties view]
+
```

Comment:
Agreed. Perhaps even a screenshot showing the Run As -> C/C++ Container Application capability. It would demonstrate potential future integrations.